__tests__/test.js

```diff
@@ -209,3 +206,22 @@ describe("PATCH PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("delete the given comment by comment_id", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .then((res) => {
+        expect(res.status).toBe(200);
+        expect(res.body[0]).toEqual({ bool: true });
+      });
+  });
+  test("return error 404 when comment_id not in DB", () => {
+    return request(app)
+      .delete("/api/comments/8998")
+      .then((res) => {
+        expect(res.status).toBe(404);
+        expect(res.text).toEqual("8998 is not a comment");
+      });
+  });
+});
```

Comment:
Missing test Status 404, non existent ID, e.g 999

Reply:
I don't think I understand, isn't that test testing returning error 404 when comment_id not in DB

Reply:
True this is already covered
You are missing a invalid ID though status 400